src/js/app.js

```diff
@@ -0,0 +1,70 @@
+import { addContent, createTable } from './dom.js';
+
+
+const addBook = addContent();
+
```

Comment:
- [ ] Here you can call `addContent()` directly instead to assign its value to a constant. This way Eslint it will stop complaining about `no-unused-vars`
`addContent();`